src/QTKit/QTCompat.cs

```diff
@@ -0,0 +1,2737 @@
+// autogenerated stub - do not edit
+#if !XAMCORE_4_0
+
+#pragma warning disable 0067
+
+namespace QTKit {
+
+	[ObjCRuntime.Obsoleted (ObjCRuntime.PlatformName.MacOSX, 10,15, message: ObjCRuntime.Constants.MacOS32bitsUnavailable)]
+	public class QTCaptureConnection : Foundation.NSObject {
+		public Foundation.NSObject GetAttribute (System.String attributeKey)
+		{
+			return default (Foundation.NSObject);
```

Comment:
Some members throw exceptions, other return default values. Is there a rule when one is chosen over the other?

Reply:
yes, it's to minimize the (IL) size. 
In many cases, like this one (and also including current manual stubs), you can't hit members throw since the type can't be instantiated (without throwing).